lib/src/zap/zap_priv.h

```diff
@@ -478,14 +478,23 @@ double zap_env_dbl(char *name, double default_value);
 zap_err_t zap_io_thread_ep_assign(zap_ep_t ep, int tpi);
 
 /**
- * Release \c ep from the zap io thread.
+ * Disable \c ep assigned to the zap io thread.
```

Comment:
The explanation on `zap_io_thread_ep_release()` and `zap_io_thread_ep_remove()` are confusing to me. Not sure how to make it better though ... maybe just saying `release` means the endpoint won't be processed by the thread any more (i.e. no more event callbacks to the application) but the endpoint still keep a reference to the thread for further stat data access, while the `remove` means disassociate the endpoint from the thread and reducing thead's ep load? I'm not sure ...